src/main/java/org/prebid/server/json/ObjectMapperProvider.java

```diff
@@ -21,7 +21,8 @@
                 .setPropertyNamingStrategy(PropertyNamingStrategy.SNAKE_CASE)
                 .setSerializationInclusion(JsonInclude.Include.NON_NULL)
                 .registerModule(new AfterburnerModule())
-                .registerModule(new ZonedDateTimeModule());
+                .registerModule(new ZonedDateTimeModule())
+                .registerModule(new TimeoutModule());
```

Comment:
Does this really required ? @rpanchyk 
This is stop failing without special module but with `@Getter` annotation